Description: improve POD:  add a NAME/abstract to ensure a correct manpage
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-01-22

--- a/lib/Geo/Coordinates/OSGB/Maps.pm
+++ b/lib/Geo/Coordinates/OSGB/Maps.pm
@@ -1148,7 +1148,11 @@
 
 =pod
 
-=head1 Data for OSGB Maps
+=head1 NAME
+
+Geo::Coordinates::OSGB::Maps - Data for OSGB Maps
+
+=head1 SUBROUTINES/METHODS
 
 This module exports no functions, but two hashes of data.
 
--- a/lib/Geo/Coordinates/OSGB/Background.pod
+++ b/lib/Geo/Coordinates/OSGB/Background.pod
@@ -1,4 +1,6 @@
-=head1 Backround and extended description
+=head1 NAME
+
+Geo::Coordinates::OSGB::Background - Background and extended description
 
 These notes are part of Geo::Coordinates::OSGB, a perl
 implementation of latitude and longitude co-ordinate conversion for
